src/components/scores/homework-cell.cjsx

```diff
@@ -71,7 +71,7 @@ HomeworkCell = React.createClass
               isConceptCoach={isConceptCoach}
               size={20}
               value={TH.getCompletedPercent(task)}
-              isLate={TH.isLate(task)}
+              isLate={TH.isDue(task)}
```

Comment:
Should this parameter be renamed from `isLate=` to `isDue=`? (can happen after all the 🔥 subsides 😄 )